CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [Earlier Releases]
+
+Releases up to and including v1.0.0 predate this changelog.
```

Comment:
This is not a blocking change, but do we have anything that describes the features here that we could point to?